fixtures/blizzard/lil-g.json

```diff
@@ -0,0 +1,400 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Lil' G",
+  "categories": ["Moving Head", "Color Changer"],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2023-09-18",
+    "lastModifyDate": "2023-09-18"
+  },
+  "links": {
+    "manual": [
+      "https://cdn.shopify.com/s/files/1/1741/5055/files/lil-g_manual_rev_a.pdf"
+    ],
+    "productPage": [
+      "https://www.blizzardpro.com/products/lil-g"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=-lwu9weWlBk"
+    ],
+    "other": [
+      "https://cdn.shopify.com/s/files/1/1741/5055/files/lil-g_cutsheet.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [201, 317, 196],
+    "weight": 3.5,
+    "power": 59,
+    "powerConnectors": {
+      "powerCON": "input only"
+    },
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "40W White LED",
+      "lumens": 723
```

Comment:
I don't see this value anywhere in the manual.  When I put 1486 lux @ 5m (9 deg beam angle) in an online calculator, I do get 723 lumens -- but 30810 lux @ 1m is more like 600 lumens.

Anyway, this seems like a reasonable value, though not a manufacturer guarantee.

Reply:
I am gonna be honest, I don't remember where I pulled that value from. I probably did use an online calculator to get to that number with the info from the manual. I try to make sure I am as thorough as I can be with the these fixture profiles